README.md

```diff
@@ -23,17 +23,54 @@
 # Swift-Kuery-ORM
 
 ## Summary
-Swift-Kuery-ORM is an Object-relational Mapping build on top of [Swift-Kuery](http://github.com/IBM-Swift/Swift-Kuery). Using Codable it provides an easy way of persisting your data.
+Swift-Kuery-ORM is an ORM (Object Relational Mapping) library built for Swift on the server. Using it allows you to simplify persistence of model objects with your server.
+
+For the sake of convenience, we've built the ORM on top of [Swift-Kuery](http://github.com/IBM-Swift/Swift-Kuery). This lets us offer developers the flexibility to customize queries made to the database, if the functionality of the ORM is insufficient.
```

Comment:
Swift-Kuery-ORM is built on top of Swift-Kuery, which means that its possible to use Swift-Kuery to customize the queries made to the database, if the functionality of the ORM is insufficient.